src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchValidationSingleSwFlowSpec.groovy

```diff
@@ -125,7 +133,8 @@ class SwitchValidationSingleSwFlowSpec extends HealthCheckSpecification {
         assumeTrue(switches as boolean, "Unable to find required switches in topology")
 
         setup: "Select a #switchType switch and retrieve default meters"
-        def sw = switches.first()
+        def sw = switches.find { it.traffGens.size() > 1 }
+        assumeTrue(sw != null) ?: "Could not find a single switch with TraffGen ports > 1"
```

Comment:
could you please use assert instead of assumeTrue?

Reply:
+